tests/packagedcode/data/rubygems/gemspec/arel.gemspec.expected.json

```diff
@@ -1,187 +1,62 @@
-    "test/nodes/test_sum.rb", 
-    "test/nodes/test_update_statement.rb", 
-    "test/support/fake_record.rb", 
-    "test/test_activerecord_compat.rb", 
-    "test/test_attributes.rb", 
-    "test/test_crud.rb", 
-    "test/test_delete_manager.rb", 
-    "test/test_factory_methods.rb", 
-    "test/test_insert_manager.rb", 
-    "test/test_select_manager.rb", 
-    "test/test_table.rb", 
-    "test/test_update_manager.rb", 
-    "test/visitors/test_depth_first.rb", 
-    "test/visitors/test_dot.rb", 
-    "test/visitors/test_ibm_db.rb", 
-    "test/visitors/test_join_sql.rb", 
-    "test/visitors/test_mssql.rb", 
-    "test/visitors/test_mysql.rb", 
-    "test/visitors/test_oracle.rb", 
-    "test/visitors/test_postgres.rb", 
-    "test/visitors/test_sqlite.rb", 
-    "test/visitors/test_to_sql.rb"
-  ], 
-  "test_files": [
-    "test/attributes/test_attribute.rb", 
-    "test/nodes/test_as.rb", 
-    "test/nodes/test_bin.rb", 
-    "test/nodes/test_count.rb", 
-    "test/nodes/test_delete_statement.rb", 
-    "test/nodes/test_equality.rb", 
-    "test/nodes/test_insert_statement.rb", 
-    "test/nodes/test_named_function.rb", 
-    "test/nodes/test_node.rb", 
-    "test/nodes/test_not.rb", 
-    "test/nodes/test_or.rb", 
-    "test/nodes/test_select_core.rb", 
-    "test/nodes/test_select_statement.rb", 
-    "test/nodes/test_sql_literal.rb", 
-    "test/nodes/test_sum.rb", 
-    "test/nodes/test_update_statement.rb", 
-    "test/test_activerecord_compat.rb", 
-    "test/test_attributes.rb", 
-    "test/test_crud.rb", 
-    "test/test_delete_manager.rb", 
-    "test/test_factory_methods.rb", 
-    "test/test_insert_manager.rb", 
-    "test/test_select_manager.rb", 
-    "test/test_table.rb", 
-    "test/test_update_manager.rb", 
-    "test/visitors/test_depth_first.rb", 
-    "test/visitors/test_dot.rb", 
-    "test/visitors/test_ibm_db.rb", 
-    "test/visitors/test_join_sql.rb", 
-    "test/visitors/test_mssql.rb", 
-    "test/visitors/test_mysql.rb", 
-    "test/visitors/test_oracle.rb", 
-    "test/visitors/test_postgres.rb", 
-    "test/visitors/test_sqlite.rb", 
-    "test/visitors/test_to_sql.rb"
-  ], 
-  "extra_rdoc_files": [
-    "History.txt", 
-    "MIT-LICENSE.txt", 
-    "Manifest.txt", 
-    "README.markdown"
-  ], 
-  "rubygems_version": "1.6.1", 
-  "required_ruby_version": ">= 0", 
-  "rubyforge_project": "arel", 
-  "loaded_from": "rubygems/arel.gemspec", 
-  "original_platform": null, 
-  "new_platform": "ruby", 
-  "specification_version": 3
-}
\ No newline at end of file
+[
+  {
+    "type": "gem",
+    "namespace": null,
+    "name": "arel",
+    "version": "2.0.7.beta.20110429111451",
+    "qualifiers": {},
+    "subpath": null,
+    "primary_language": "Ruby",
+    "description": "Arel is a SQL AST manager for Ruby. It",
+    "release_date": null,
+    "parties": [
+      {
+        "type": "person",
+        "role": "author",
+        "name": "Aaron Patterson, Bryan Halmkamp, Emilio Tagua, Nick Kallen",
```

Comment:
These were originally lists of values for authors and emails in https://github.com/nexB/scancode-toolkit/blob/157a64701f996676f7b252a514760bc9668c68aa/tests/packagedcode/data/rubygems/gemspec/arel.gemspec#L8
and in https://github.com/nexB/scancode-toolkit/pull/2075/files#diff-f2830f25610f80643082bfc08bb4ca40L9
why losing that?

Reply:
It is not fixed that number of authors and emails are same. So I both emails and authors in string.

Reply:
As suggested above, create instead as many Party as needed in this case.